app/admin/scholarship_application.rb

```diff
@@ -0,0 +1,35 @@
+ActiveAdmin.register ScholarshipApplication do
+  permit_params :reason, :terms_accepted, :scholarship_id, :user_id
+  index do
+    selectable_column
+    id_column
+
+    column 'Scholarship' do |scholarship_app|
+      scholarship_app.scholarship.name
+    end
+
+    column 'User' do |user_id|
+      user_id.user.email
```

Comment:
Can you please create a method in `app/models/user.rb` for a user's full name that returns of string of their first and last name, and then use that method here, in place of `email`.